response.go

```diff
@@ -200,6 +213,7 @@ func NewResponse(status, headers, body string) (*Response, error) {
 	}, nil
 }
 
+// Write satisfies interface io.Writer.
```

Comment:
Actually it doesn't:
```golang
type Writer interface {
        Write(p []byte) (n int, err error)
}
```
I think the key here is that Write sends the Response over the wire

Reply:
My mistake. How did i overlook io.Writer's signature?